backend/routes.js

```diff
@@ -369,3 +369,63 @@ route.POST('/zkpp/updatePasswordHash/{contract}', {
 
   return Boom.internal('internal error')
 })
+
+// PWA push notification
+
+route.GET('/push/publickey', {}, function () {
+  return VAPID_PUBLIC_KEY
```

Comment:
It'd be nice to add the headers for this, such as `content-type`, etc. I've looked and unfortunately there doesn't seem to be a standard URL for this, so this path is fine (otherwise, I'd have suggested using the standard path).

Reply:
@corrideat Good point. updated accordingly.